src/styles/blocks/card.scss

```diff
@@ -0,0 +1,56 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-width;
+  background: $secondary-backgroud-color;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__photo {
+    height: 134px;
+    padding: 0 3px 40px;
```

Comment:
use flex properties on flex container to align elements inside, not padding with magic numbers